Sources/SpeziAccount/AccountSetup.swift

```diff
@@ -115,7 +115,8 @@ public struct AccountSetup<Header: View, Continue: View>: View {
         }
             .onChange(of: account.signedIn) {
                 if let details = account.details, case .setupShown = setupState {
-                    let missingKeys = account.configuration.missingRequiredKeys(for: details, includeCollected: details.isNewUser)
+                    // set includeCollected to true to collect the same information through self sign-up and administered signup
+                    let missingKeys = account.configuration.missingRequiredKeys(for: details, includeCollected: true)
```

Comment:
Wouldn't this ask the user to supply optional information every-time they sign in with their account again?
E.g., user signs up, normally (e.g., within the app), doesn't supply the optional `dateOfBirth` (which is fine) but supplies all required information. They eventually log out of the app and back in at a later point in time. This code would then re-request the user to supply their `dateOfBirth`, or am I missing something?